### ISSUE TITLE
Get a grip on sensibility

### ISSUE THREAD

**Issue by apocatarsis** [apocatarsis/Naga_KeypadMapper#3]
Synapse lets you assign a button to change the sensibility of the mouse. This would be neat!!
But I do not know the first thing about it

**Comment by drunkscientist** [comment]
so assuming you mean 'sensitivity' i found some relevant information but dont know enough about programming to do anything useful with it 

http://askubuntu.com/questions/205676/how-to-change-mouse-speed-sensitivity

http://unix.stackexchange.com/questions/90572/how-can-i-set-mouse-sensitivity-not-just-mouse-acceleration

**Comment by Destroyer** [comment]
I don't think we need to implement this at all since this exists for DPI change https://github.com/mbuesch/razer

**Comment by apocatarsis** [comment]
But that is super big! It should work without any more work using this project alone!

I was thinking of something like in [here](http://unix.stackexchange.com/questions/90572/how-can-i-set-mouse-sensitivity-not-just-mouse-acceleration) or [here.](http://askubuntu.com/questions/205676/how-to-change-mouse-speed-sensitivity)

Something like adding a new option to mapping.txt like dpistep=-300 so you can bind a button to increase/decrease sensitivity. Instead of xdotool we culd use some xinput magic. I really think this would be neat!.

**Comment by mrlinuxfish** [comment]
I was able to change the speed at which the pointer moves by modifying the ConstantDeceleration in the xorg config for the mouse. I think this can also be achieved through xset m [acceleration ratio] [acceleration threshold] (https://wiki.archlinux.org/index.php/Mouse_acceleration).

**Comment by RaulPPelaez** [comment]
I was thinking more of as in changing the mouse DPI, like you do with synapse on windows. I dont know if that would be equivalent, but my guess is not. Let me know if you find out!

**Comment by apocatarsis** [comment]
I was thinking more of as in changing the mouse DPI, like you do with synapse on windows. I dont know if that would be equivalent, but my guess is not. Let me know if you find out!

**Comment by TheGuardianWolf** [comment]
There's a command to do this

```
xinput set-prop <device_id> "Coordinate Transformation Matrix" x 0 0 0 x 0 0 0 x
```

Replace X with a floating point number.
